omim2obo/main.py

```diff
@@ -231,10 +225,14 @@ def run(use_cache: bool = False):
             graph.add((OMIM[mim_number], RDFS.subClassOf, OMIMPS[ps_id]))
 
     # Morbid map (cyto locations)
-    morbid_map: Dict[str, List[str]] = parse_morbid_map(
-        retrieve_mim_file('morbidmap.txt', download_files_tf))
-    for mim_number in morbid_map:
-        phenotype_mim_number, cyto_location = morbid_map[mim_number]
+    morbid_map: Dict[str, Dict] = parse_morbid_map(get_mim_file('morbidmap.txt', download_files_tf))
+    for mim_number, mim_data in morbid_map.items():
+        # todo?: unused `mim_data` keys:
```

Comment:
```py
        # todo?: unused `mim_data` keys:
        #   'gene_mim_number': Should this be used somewhere?
        #   'gene_symbols':  Should this be used somewhere?
        #   'phenotype_label': I think this already added to graph because the MIM# and label shared in other file(s)
```

@matentzn `morbidmap.txt` has _phenotype/disorder MIM #'s_, and also _gene MIM#'s_. However, it looks like we have not been doing anything with the gene MIM number. I'm wondering if maybe Dazhi didn't do anything with this because, similar to the phenotype label, we're already extracting the MIM#, label, and information from one of the other files. I guess I could do with some double checking on this.

Now that I'm thinking about it, perhaps that's why Dazhi didn't do anything with the _gene symbols_ here as well. Perhaps that's because this is already being handled by the parsing of `mim2gene.txt` or elsewhere.

Reply:
We spoke today and it looks like things are fine for now. This is probably correct, that this data is being fetched elsewhere.